cmd/insprd/memory/tree/dapp_utils.go

```diff
@@ -302,3 +306,22 @@ func (amm *AppMemoryManager) recursiveBoundaryValidation(app *meta.App) error {
 
 	return nil
 }
+
+// SelectBrokerFromPriorityList takes a broker priority list and returns the first
+// broker that is available
+func SelectBrokerFromPriorityList(brokerList []string) string {
+	bmm := brokers.GetBrokerMemory()
```

Comment:
this is a direct access to a singleton, it should use the access point that is soon to be available on insprd's Server struct.